WordPressKit/Insights/StatsAllTimesInsight.swift

```diff
@@ -0,0 +1,39 @@
+public struct StatsAllTimesInsight {
+    public let postsCount: Int
+    public let viewsCount: Int
+    public let bestViewsDay: Date
+    public let visitorsCount: Int
+    public let bestViewsPerDayCount: Int
+}
+
+
+extension StatsAllTimesInsight: InsightProtocol {
+
+    //MARK: - InsightProtocol Conformance
+    public init?(jsonDictionary: [String: AnyObject]) {
+        guard
+            let statsDict = jsonDictionary["stats"] as? [String: AnyObject],
```

Comment:
I personally like this style for `guard/let` syntax.

Reply:
:)